src/plugins/content/accessibility/keyNavigation/strategies/itemNavigation.js

```diff
@@ -51,6 +54,18 @@ const removeLabelledByAttribute = cursor => {
     $element.removeAttr('aria-labelledby', '');
 };
 
+/**
+ * Adds attributes on navigation focus and blur
+ * 
+ * @param {Navigator} navigator 
+ */
+const manageLabelledByAttribute = (navigator) => {
+    if (navigator) {
+        navigator.on('focus', addLabelledByAttribute);
+        navigator.on('blur', removeLabelledByAttribute); // applies WCAG behavior for the radio buttons
```

Comment:
not sure this relates to the WCAG behavior for the radio buttons, but this is more related to labelling.
Anyway, not a big issue